src/SFA.DAS.EAS.Web/Views/Transfers/TransferDetails.cshtml

```diff
@@ -100,16 +91,15 @@
                 </div>
             </fieldset>
             <button type="submit" class="button" id="submit-transfer-connection">Continue</button>
-            <a class="button-link" href="@Url.Action("Index", "EmployerCommitments")" aria-label="Cancel">Cancel</a>
```

Comment:
I can't put this comment any closer to the offending markup, but I think:
@Html.Partial("ValidationSummary", Html.ViewData.ModelState)
is in the wrong place. It should appear at the top of the screen.